### PR TITLE
#43 remove unnecessary EcoSystem argument from upgrade

### LINKED ISSUE #43: EcoSystem.update() has unnecessary argument of type EcoSystem
Since `update()` is a method, it does not need to take `EcoSystem` as an argument. It can just rely on `this`.

### PR DESCRIPTION
**Pull request by jelemux**
Resolves #43

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Removed
+- [#43] Argument of type `EcoSystem` from `update()`-method
+### Added
```
Comment (ppxl) [inline CHANGELOG.md:10]: We should have a new line between the H3 headers
```suggestion

### Added
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!groovy
 
 // Keep the version in sync with the one used in pom.xml in order to get correct syntax completion.
-@Library('github.com/cloudogu/ces-build-lib@63442716')
+@Library('github.com/cloudogu/ces-build-lib@1.67.0')
```
Comment (ppxl) [inline Jenkinsfile:4]: :blue_heart:

File: src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy [hunk 3]
```diff
@@ -213,6 +224,14 @@ class EcoSystem {
         }
     }
 
+    /**
+     * Installs the given dogu and performs an upgrade on it.
+     * Before installation, the dogu is purged to ensure a green slate.
```
Comment (ppxl) [inline src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy:229]: :+1: For doc comments.

But: Did you mean `a clean slate`?

**Review by ppxl — CHANGES_REQUESTED** [review]
LGTM, just that idiom is all that would ask to change

File: src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy (hunk 3 above)
Comment (jelemux) [reply src/com/cloudogu/ces/dogubuildlib/EcoSystem.groovy:229]: Oops, I knew something was off about that but I couldn't quite figure it out.

**Review by ppxl — APPROVED** [review]
LGTM

**Pull request merged**